components/dashboard/src/components/RepositoryFinder.tsx

```diff
@@ -87,15 +92,28 @@ export default function RepositoryFinder({
         // searchString ignore here as list is already pre-filtered against it
         // w/ mirrored state via useUnifiedRepositorySearch
         (searchString: string) => {
-            return repos.map((repo) => {
+            const result = repos.map((repo) => {
                 return {
                     id: repo.projectId || repo.url,
                     element: <SuggestedRepositoryOption repo={repo} />,
                     isSelectable: true,
                 } as ComboboxElement;
             });
+            if (hasMore) {
+                // add an element that tells the user to refince the search
+                result.push({
+                    id: "more",
+                    element: (
+                        <div className="text-sm text-gray-400 dark:text-gray-500">
+                            There are more elements that are not displayed. Please refine your search.
```

Comment:
Seems that maybe "results" would fit better in here. I was a bit confused about what "elements" meant at first.

```suggestion
                            There are more results that are not displayed. Please refine your search.
```